LukeTHardy/gardenproject
# Lh fixtures

## Overview:

- Added and tested final fixture, plantcritterpairings

## Testing:

- Start the server and client app
- Login and navigate to any PlantDetails or CritterDetails page
- Potential Critters and May Be Found On fields should either be populated with plant/critter names, or say "None to show" or "All Plants" respectively
